app/views/MediaAutoDownloadView/ListPicker.tsx

```diff
@@ -61,9 +81,18 @@ const ListPicker = ({
 
 	return (
 		<List.Item
-			title={title}
 			onPress={() => showActionSheet({ options: getOptions() })}
-			right={() => <Text style={[styles.title, { color: colors.fontHint }]}>{label}</Text>}
+			title={() => (
+				<View style={styles.leftTitleContainer}>
+					<Text style={[styles.leftTitle, { color: colors.fontDefault }]}>{title}</Text>
+				</View>
+			)}
+			right={() => (
+				<View style={styles.rightTitleContainer}>
+					<Text style={[styles.rightTitle, { color: colors.fontHint }]}>{label}</Text>
+				</View>
+			)}
+			rightContainerStyle={styles.rightContainer}
```

Comment:
I agree with the changes, but I think the use case grow up in size, so I prefer to officialize this ListPicker as part of `containers/List` and add stories.

Reply:
Agreed